Realm/Realm/DatabaseTypes/RealmCollectionBase.cs

```diff
@@ -189,15 +190,41 @@ public IRealmCollection<T> Freeze()
             return CreateCollection(frozenRealm, frozenHandle);
         }
 
-        public IDisposable SubscribeForNotifications(NotificationCallbackDelegate<T> callback)
-            => SubscribeForNotificationsImpl(callback, shallow: false);
+        public IDisposable SubscribeForNotifications(NotificationCallbackDelegate<T> callback, KeyPathsCollection? keyPathCollection = null)
```

Comment:
```suggestion
        public IDisposable SubscribeForNotifications(NotificationCallbackDelegate<T> callback, KeyPathsCollection? keyPathsCollection = null)
```

No strong opinion about this, just bugs me that it's different between methods.